src/parsing/index.ts

```diff
@@ -22,15 +22,17 @@ export interface ParseResult {
     suggestColumnAliases?: ColumnAliasSuggestion[];
     suggestCommonTableExpressions?: unknown;
     suggestDatabases?: DatabasesSuggestion;
-    suggestFilters?: unknown;
-    suggestFunctions?: unknown;
+    suggestFilters?: FiltersSuggestion;
+    suggestFunctions?: FunctionsSuggestion;
+    suggestValues?: ValuesSuggestion;
     suggestGroupBys?: unknown;
     suggestIdentifiers?: IdentifierSuggestion[];
     suggestTemplates?: boolean;
     suggestEngines?: {
         engines: Engines;
         functionalEngines: Engines;
     };
+    colRef?: ColRef;
```

Comment:
Let's not use shortened variable names in our types. Please name it `ColumnReference`.